components/actiongroup/stories/template.js

```diff
@@ -119,6 +119,7 @@ export const OverflowOption = (context) => Container({
 						},
 						{
 							iconName: "More",
+							label: "More options",
```

Comment:
In my screenreader testing, this button was just announced as "toggle button." Do we have a specific label for a button like this, that probably triggers a popover/menu?